src/Admin/AdminExtensionInterface.php

```diff
@@ -75,7 +75,7 @@ public function configureRoutes(AdminInterface $admin, RouteCollection $collecti
      * @return void
      *
      * @phpstan-param AdminInterface<T> $admin
-     * @phpstan-param AdminInterface<T>|null $childAdmin
+     * @phpstan-param AdminInterface<object>|null $childAdmin
```

Comment:
The admin and child admin can mange different types, so `object` is more accurate here.